src/app/public/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -61,6 +66,9 @@ const SKY_AUTONUMERIC_VALIDATOR = {
 })
 export class SkyAutonumericDirective implements OnInit, ControlValueAccessor, Validator {
 
+  /**
+   * Assigns the name of a property from `SkyAutonumericOptionsProvider`.
```

Comment:
I'm not sure this is correct for this version of autoNumeric. This input allows consumers to pass options to the underlying autoNumeric utility.